From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Vinayak Menon <vinmenon@codeaurora.org>
Date: Wed, 28 Oct 2020 11:42:24 +0530
Subject: ANDROID: gki_defconfig: Enable CONFIG_PAGE_OWNER

This helps enable the page owner feature at runtime via
kernel param, and helps in memory accouting and leak
debugging. Enabling just this config does not incur any
significant overhead.
Explicit CONFIG_DEBUG_FS=y is removed because CONFIG_PAGE_OWNER
enables it implicitly.

Bug: 171354330
Change-Id: I103c0276a01ef6fa0673d88c4a8678b31af2b8a2
Signed-off-by: Vinayak Menon <vinmenon@codeaurora.org>
Signed-off-by: Suren Baghdasaryan <surenb@google.com>
---
 arch/arm64/configs/gki_defconfig | 2 +-
 arch/x86/configs/gki_defconfig   | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/configs/gki_defconfig b/arch/arm64/configs/gki_defconfig
--- a/arch/arm64/configs/gki_defconfig
+++ b/arch/arm64/configs/gki_defconfig
@@ -573,7 +573,7 @@ CONFIG_DEBUG_INFO=y
 CONFIG_HEADERS_INSTALL=y
 # CONFIG_SECTION_MISMATCH_WARN_ONLY is not set
 CONFIG_MAGIC_SYSRQ=y
-CONFIG_DEBUG_FS=y
+CONFIG_PAGE_OWNER=y
 CONFIG_DEBUG_STACK_USAGE=y
 CONFIG_DEBUG_MEMORY_INIT=y
 CONFIG_PANIC_ON_OOPS=y
diff --git a/arch/x86/configs/gki_defconfig b/arch/x86/configs/gki_defconfig
--- a/arch/x86/configs/gki_defconfig
+++ b/arch/x86/configs/gki_defconfig
@@ -516,7 +516,7 @@ CONFIG_DEBUG_INFO=y
 CONFIG_HEADERS_INSTALL=y
 # CONFIG_SECTION_MISMATCH_WARN_ONLY is not set
 CONFIG_MAGIC_SYSRQ=y
-CONFIG_DEBUG_FS=y
+CONFIG_PAGE_OWNER=y
 CONFIG_DEBUG_STACK_USAGE=y
 CONFIG_DEBUG_MEMORY_INIT=y
 CONFIG_PANIC_ON_OOPS=y
